Update/_meak_020.txt

```diff
@@ -4504,15 +4507,17 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f3ceb3>喜一郎</color>", NULL, "<color=#f3ceb3>Kiichiro</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 18, "ps3/s06/18/153300132", 128, TRUE);
 	OutputLine(NULL, "「詩音ちゃんだって、建設省の幹部の息子…、",
-		   NULL, "\"You've heard about", Line_WaitForInput);
+		   NULL, "\"You've heard about", Line_ContinueAfterTyping);
 	OutputLine(NULL, "というか建設大臣の孫だな…。",
 		   NULL, " the kidnapping case of", Line_ContinueAfterTyping);
 	SetValidityOfInput( FALSE );
 	Wait( 1000 );
 	SetValidityOfInput( TRUE );
-	ModPlayVoiceLS(4, 18, "ps3/s06/18/153300133", 128, TRUE);
 	OutputLine(NULL, "あれの誘拐事件の話は知ってるだろ…？」",
-		   NULL, " the grandson of the Minister of construction, right?\"", GetGlobalFlag(GLinemodeSp));
+		   NULL, " the grandson of the Minister of construction, right?\"", Line_Continue);
+	WaitToFinishVoicePlaying(4);
+	ModPlayVoiceLS(4, 18, "ps3/s06/18/153300133", 128, TRUE);
```

Comment:
In cases like this put both `WaitToFinishVoicePlaying` and `ModPlayVoiceLS` before the `OutputLine`. Revert the `GetGlobalFlag(GLinemodeSp)` too.

That one case when I put it after was an edge-case and the solution is not ideal either. Don't do that.

Reply:
I must've been tired to not notice this solution, heh. It's fixed and will be added with tomorrow's changes